githubpoxy.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"io"
+	"time"
+
+	"github.com/go-rod/rod"
+)
+
+type GitHubProvider struct {
+}
+
+func (ghp GitHubProvider) Login(login, password string) error {
+	const loginURL = "https://github.com/login"

```

Comment:
you're gonna get some good points with @ARolek for declaring this here as a `const`